releasenotes/notes/fix-fid_statevector_kernel-pickling-b7fa2b13a15ec9c6.yaml

```diff
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Fixed a bug where :class:`~qiskit_machine_learning.kernels.FidelityQuantumKernel` threw an error when pickled. This was due 
```

Comment:
Just as a note you can do this too, as long as its a class that is unique ''':class:`.FidelityQuantumKernel` ``` where with the `.` sphinx will find the class without you having to type the full thing. 

Unless any of how it was fixed affects the end user then just stating its fixed and can be pickled as expected now suffices. The release notes are really just the public API and changes/fixes perceived when using the API for end-users - internal details of how it was fixed, unless they have additional impacts in some way are not really relevant to such users. Eg. to fix things we could have stopped user lru_cache and done something else internally. All the end user needs to know is that the problem is fixed. Hence to me only the content of the first sentence is needed here for a release note for this.

Reply:
Ah, okay I will reduce it down to the first sentence.